provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -187,16 +187,32 @@ def _create_profile_descriptions(
 
         return profiles
 
+    def _create_fdmi_filter_descriptions(
+            self, nodes: List[NodeDesc]) -> List[FdmiFilterDesc]:
+        fdmi_filters: List[FdmiFilterDesc] = []
+
+        fdmi_filters.append(
```

Comment:
These values don't look like a harmless stub. Per my understanding, this will register non empty FDMI filter that will be applied unconditionally. Given it is CDF generator, this filter will be added every time Mini provisioner deploys Hare.

Is it correct?

Reply:
Yes, it is correct. I tried to add an empty filters section, but seems like `cdf.py` doesn't support this and I can't avoid adding it because of how dhall works. Would you please help me with this?

Reply:
@max-seagate I've just pushed a fixup commit to your PR. Please have a look and squash if it looks good.

Reply:
Thank you. Let me check.

Reply:
Single node bootstrap succeeds with your patch, so seems like it's good. Thanks.